kubernetes-sigs/controller-runtime
# Release for v0.5.1

```
# v0.5.1

## :sparkles: New Features

- Add controllerutil.EnsureOwnerReference (#816)
- Take context when getting informer. (#663)
- Support restconfig to target the control plane (#756)
- Create a supported way of setting a cache on source.Kind (#794)
- Webhook support in envtest (#787)
- Enable webhook server set-up mTLS service to verify client's certificate (#802)
- Make workqueue rate limiter configurable during controller creation (#731)

## :bug: Bug Fixes

- Fix testdata imports in pkg/webhook/conversion (#843)
- Respect context in unstructured client (#812)
- Fix informer cache creating pointers to pointers (#796)
- Use GenerateName during fake client's Create method (#815)

*Thanks to all our contributors!*
```
